docs/review_app_setup.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+
+# Description: This is a script to help automate the process of building
```

Comment:
I think this can be more declarative, "Run this script to automate...". I'd also add a comment with the command to run it, to make this accessible as possible. I'm assuming a user will have to pass the name variable.

Reply:
+1 - would document the invocation with name as required argument